### DISCUSSION TITLE
Special meeting via stream

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1724] [Q&A] [answered]
For a live stream, for example: **meeting via stream (browser)**, what are the options to use M3 and stream the stream (browser) screen and share the video to KH+Zoom?
🎦

**Comment by sircharlo** [comment, marked as answer]
For such a program, wouldn't the simplest option be to connect using a normal web browser and share using Zoom screen sharing? Why would M3 be needed?

**Reply by joaochris** [reply]
Or you can also use OBS and add a browser font. I tested it and it works.

**Reply by e-Feitosa** [reply]
I liked the suggestions, @joaochris.  I'm going to test this via OBS Studio. @sircharlo the question asked was due to knowing several brothers who use the M3, and who would have difficulty in this transmission and the more options we have, the live transmission will be easier. Many thanks for the suggestions. It's just that I thought of something like accessing the stream.jw.org site via M3, just like the beta version to access the jw.org site, but I saw that I couldn't type the address in the navigation bar.
